src/fragments/lib/push-notifications/common/getting_started/cross-platform-prereq.mdx

```diff
@@ -31,15 +31,5 @@ import fcmPreReq from '/src/fragments/lib/push-notifications/android/getting_sta
 
 <Fragments fragments={{ flutter: fcmPreReq, 'react-native': fcmPreReq }} />
 
-### Applying the Google services plugin
```

Comment:
These instructions repeat what is already described in the Firebase docs and make it seem like there is another step beyond Setting up FCM section.  Furthermore they are incomplete as the root and app level `build.gradle` needs to be updated.